scss/bootstrap/vendor/select2/select2-bootstrap.scss

```diff
@@ -1,94 +1,257 @@
 .form-control .select2-choice {
-    border: 0;
-    border-radius: 2px;
-    text-decoration: none !important;
+  border: 0;
+  border-radius: 2px;
+  text-decoration: none !important;
 }
 
 .form-control .select2-choice .select2-arrow {
-    border-radius: 0 2px 2px 0;
-    background: transparent;
-    border-left: 0 none;
-    right: 4px;
-    top: 2px;
+  border-radius: 0 2px 2px 0;
+  background: transparent;
+  border-left: 0 none;
+  right: 4px;
+  top: 2px;
 }
 
-.form-control.select2-container {
-    padding: 0;
-    height: auto;
+.form-control.select2-container,
+.form-group .select2-container {
+  padding: 0;
+  height: auto;
+  background: $crm-white;
+  border: 1px solid $crm-darker-grayblue;
+  box-shadow: $crm-form-control-shadow;
+  height: 30px;
 }
 
-.form-control.select2-container.select2-dropdown-open {
-    border-color: #5897FB;
-    border-radius: 3px 3px 0 0;
+.form-control.select2-container.select2-dropdown-open,
+.form-group .select2-container.select2-dropdown-open {
+  border-color: $crm-select2-dropdown-border;
+  border-radius: 3px 3px 0 0;
 }
 
-.form-control .select2-container.select2-dropdown-open .select2-choices {
-    border-radius: 3px 3px 0 0;
```

Comment:
Are you sure about space between .form-control and .select2-container? I see there is no space in below and above code